frontend/src/app/libs/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { IUser } from '../container/models/user';
+import { SessionFacadeService } from '../container/store/session/session.facade';
+
+@Component({
+	selector: 'app-sign-in',
+	templateUrl: './sign-in.component.html',
+	styleUrls: ['./sign-in.component.scss'],
+})
+export class SignInComponent {
+	public user: IUser;
```

Comment:
Naming interfaces with starting 'I' is .net and java practice, please rename to `User` interface.